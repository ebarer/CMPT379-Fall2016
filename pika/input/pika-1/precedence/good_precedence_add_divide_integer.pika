exec {

  print 50 + 91 / 69, _n_.
  print 63 + 42 / 29, _n_.
  print 35 + 32 / 44, _n_.
  print 35 + 62 / 31, _n_.
  print 37 + 91 / 31, _n_.
  print 75 + 3 / 81, _n_.
  print 73 + 23 / 77, _n_.
  print 73 + 63 / 34, _n_.
  print 1 + 99 / 14, _n_.
  print 11 + 40 / 61, _n_.
  print 6 + 82 / 48, _n_.
  print 94 + 71 / 35, _n_.

  print 28 / 27 + 78, _n_.
  print 24 / 68 + 66, _n_.
  print 77 / 82 + 2, _n_.
  print 54 / 49 + 19, _n_.
  print 88 / 74 + 76, _n_.
  print 6 / 56 + 16, _n_.
  print 61 / 79 + 7, _n_.
  print 89 / 49 + 13, _n_.
  print 39 / 31 + 92, _n_.
  print 68 / 47 + 92, _n_.
  print 23 / 9 + 98, _n_.
  print 43 / 8 + 2, _n_.

  print 59 + 15 / 77 + 87, _n_.
  print 27 + 12 / 29 + 93, _n_.
  print 17 + 43 / 60 + 14, _n_.
  print 36 + 80 / 26 + 1, _n_.
  print 70 + 16 / 46 + 85, _n_.
  print 29 + 22 / 97 + 54, _n_.
  print 16 + 8 / 34 + 50, _n_.
  print 97 + 44 / 37 + 37, _n_.
  print 84 + 76 / 36 + 63, _n_.
  print 81 + 17 / 6 + 26, _n_.
  print 35 + 80 / 20 + 86, _n_.
  print 92 + 17 / 8 + 4, _n_.

  print 53 / 30 + 81 / 81, _n_.
  print 13 / 54 + 89 / 38, _n_.
  print 43 / 11 + 53 / 36, _n_.
  print 30 / 15 + 92 / 56, _n_.
  print 53 / 21 + 46 / 34, _n_.
  print 18 / 21 + 86 / 63, _n_.
  print 8 / 54 + 73 / 43, _n_.
  print 24 / 82 + 50 / 57, _n_.
  print 69 / 59 + 63 / 49, _n_.
  print 69 / 4 + 20 / 57, _n_.
  print 28 / 37 + 16 / 62, _n_.
  print 68 / 40 + 90 / 80, _n_.

}

