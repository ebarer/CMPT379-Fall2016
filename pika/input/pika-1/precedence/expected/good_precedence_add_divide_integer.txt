51
64
35
37
39
75
73
74
8
11
7
96
79
66
2
20
77
16
7
14
93
93
100
7
146
120
31
40
155
83
66
135
149
109
125
98
2
2
4
3
3
1
1
0
2
17
0
2
